src/components/input/mockFunctions.ts

```diff
@@ -0,0 +1,85 @@
+export function getRandomDelayInSeconds(maxSeconds: number) {
+  const conversionRate = 1000
+  const minDelay = 1000
+  const maxDelay = maxSeconds * conversionRate
+  return Math.floor(Math.random() * (maxDelay - minDelay + 1)) + minDelay
+}
+
+export function onFileDelete(fileId: string): Promise<{ isDeleted: boolean }> {
+  return new Promise((resolve) => {
+    setTimeout(() => {
+      resolve({ isDeleted: true })
+      // eslint-disable-next-line no-console
+      console.log('File deleted:', fileId)
```

Comment:
I think we can remove the console log.
I notice that this file is being used in storybook as well as tests. Since it is not required for the final published library, we should exclude it from the production build files.